docs/examples/cloudstoragesource/README.md

```diff
@@ -12,12 +12,51 @@ Object Notifications for when a new object is added to Google Cloud Storage
 
 1. [Create a Pub/Sub enabled Service Account](../../install/pubsub-service-account.md)
 
-1. Enable the `Cloud Storage API` on your project:
-
-   ```shell
-   gcloud services enable storage-component.googleapis.com
-   gcloud services enable storage-api.googleapis.com
-   ```
+1. Enable the `Cloud Storage API` on your project and give Google Cloud Storage
+   permissions to publish to GCP Pub/Sub. Currently, we support two methods:
+   automated script or manual
+   
+   - Option 1(via automated script): 
```

Comment:
Mark one option as recommended. It helps guide the readers.

Reply:
I didn't see which option is recommended...

Reply:
This is not specific to this PR, but I think in general our installation docs include too much information with many levels of options. They can be intimidating to new users. Maybe we should document the recommended path in the main doc, and link details separately for users who want to tweak their installation.

But to do that we first need to have a good opinionated  recommendation :) We can follow up later.